eip-5478.md

```diff
@@ -0,0 +1,45 @@
+---
+eip: 5478
+title: Reducing the gas cost of contract creation with existing code
+author: Qi Zhou (@qizhou)
+discussions-to: https://ethereum-magicians.org/t/eip-5478-reducing-the-gas-cost-of-contract-creation-with-existing-code/10419
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-08-17
+requires: 1014, 2929
+---
+
+## Abstract
+
+Adding a new opcode, `CREATE3`, that is identical to `CREATE2` but with potentially much lower gas cost by accepting an additional argument `existing_contract_address` that already stored the code of the new contract.
```

Comment:
What do you think of using a more descriptive name, like, for example, `CREATECOPY`?

Reply:
Sound good!  I would like to use `CREATE2COPY` because this is the copy version for `CREATE2`, but we may `CREATECOPY` as the copy version for `CREATE` in the future.